Purchases/Networking/RCHTTPClient.m

```diff
@@ -88,6 +89,12 @@ - (void)performRequest:(NSString *)httpMethod
                                                                path:path
                                                         requestBody:requestBody
                                                             headers:defaultHeaders];
+    if (!urlRequest) {
+        RCErrorLog(@"Could not create request to %@ with body %@", path, requestBody);
+        completionHandler(-1,
+                          nil,
+                          [RCPurchasesErrorUtils networkErrorWithUnderlyingError:RCPurchasesErrorUtils.unknownError]);
+    }
```

Comment:
there might be a nicer-looking way of doing this. But this is the essence of the PR